l10n_co_res_partner/models/ciiu.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#                                                                             #
+# Copyright (C) 2016  Dominic Krimmer                                         #
+#                                                                             #
+# This program is free software: you can redistribute it and/or modify        #
+# it under the terms of the GNU Affero General Public License as published by #
+# the Free Software Foundation, either version 3 of the License, or           #
+# (at your option) any later version.                                         #
+#                                                                             #
+# This program is distributed in the hope that it will be useful,             #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of              #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the               #
+# GNU Affero General Public License for more details.                         #
+#                                                                             #
+# You should have received a copy of the GNU Affero General Public License    #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.       #
+###############################################################################
+
+from openerp import models, fields, api
+
+
+class IndustrialClassification(models.Model):
+    _name = "ciiu"  # res.co.ciiu
+    _description = "ISIC List"
+
+    name = fields.Char(
+        string="Code and Description",
+        store=True,
+        compute="_compute_concat_name"
+    )
```

Comment:
Name should be a regular field. Use `name_get` instead to control how it is displayed.